### PR TITLE
Description, help text and readme updated

### PR DESCRIPTION
**Pull request by nishubansal**

### CODE REVIEW

File: Extensions/ServiceNow/Src/vss-extension.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
   "name": "ServiceNow change request management",
   "publisher": "ms-devlabs",
```
Comment (bansalaseem) [inline Extensions/ServiceNow/Src/vss-extension.json]: should not the publisher be microsoft?

File: Extensions/ServiceNow/Src/readme.md [hunk 2]
```diff
@@ -1 +1,58 @@
...
+**Gate Success Criteria** :
+- **Desired state**: The gate would succeed and the pipeline continues when the change request status is same as the provided value.
+
+4. Add a task to update the status of the change
+
+![Update task](images/agentless_task.png)
+
+**Inputs for Update change request task**:
+
+- **Change request number**: Number of the change request that you want to update.
```
Comment (bansalaseem) [inline Extensions/ServiceNow/Src/readme.md]: We should talk about the variable (PreDeployGate)….. here

File: Extensions/ServiceNow/Src/Tasks/CreateAndQueryChangeRequest/task.json [hunk 3]
```diff
@@ -18,7 +17,7 @@
     "Minor": 1,
     "Patch": 0
   },
-  "instanceNameFormat": "ServiceNow Change Management",
+  "instanceNameFormat": "ServiceNow change management",
   "groups": [
     {
       "name": "schedule",
```
Comment (shashban) [inline Extensions/ServiceNow/Src/Tasks/CreateAndQueryChangeRequest/task.json]: Is this an appropriate name?

File: Extensions/ServiceNow/Src/Tasks/UpdateChangeRequest/task.json [hunk 4]
```diff
@@ -1,12 +1,11 @@
 {
-  "id": "6BBE8736-EE83-4BBF-A0AD-96B118797CC4",
+  "id": "37AC13CA-AEAD-4E19-A5AC-E5366051FC1C",
   "name": "UpdateServiceNowChangeRequest",
   "friendlyName": "Update ServiceNow Change Request",
   "description": "Update ServiceNow Change Request",
```
Comment (shashban) [inline Extensions/ServiceNow/Src/Tasks/UpdateChangeRequest/task.json]:
```suggestion
  "description": "Update ServiceNow change request",
```

File: Extensions/ServiceNow/Src/readme.md [hunk 5]
```diff
@@ -1 +1,58 @@
...
\ No newline at end of file
+
+# ServiceNow Change Management Extension
+
+ServiceNow is a software-as-a-service (SaaS) provider of IT service management (ITSM) software, including change management.
+Specific change management subprocesses include change risk assessment, change scheduling, change approvals and oversight. 
+With change management, your organization can reduce the risks associated with change, while speeding up the deployments with Azure pipelines. 
+
+This extension enables integration of ServiceNow change management with Azure pipelines.
+It includes a [release gate](https://docs.microsoft.com/en-us/azure/devops/pipelines/release/approvals/gates?view=vsts) to create a change request in ServiceNow and hold the pipeline till for the change management process to signal implementation.
```
Comment (shashban) [inline Extensions/ServiceNow/Src/readme.md]: missing newline.

File: Extensions/ServiceNow/Src/readme.md [hunk 6]
```diff
@@ -1 +1,58 @@
...
+An agentless task to close (update state of) the change request after the deployment is also provided.
```
Comment (shashban) [inline Extensions/ServiceNow/Src/readme.md]: Newline

File: Extensions/ServiceNow/Src/readme.md [hunk 7]
```diff
@@ -1 +1,58 @@
...
+
+The deployment process in Azure pipelines helps in automation of the deployment and complement the controls offered by ServiceNow.
+
+## How to use the integration
+1. The integration requires the Azure DevOps application to be installed on the ServiceNow instance.
```
Comment (shashban) [inline Extensions/ServiceNow/Src/readme.md]: please include the name of the Application as submitted to ServiceNow

File: Extensions/ServiceNow/Src/vss-extension.json (hunk 1 above)
Comment (shashban) [reply Extensions/ServiceNow/Src/vss-extension.json]: Should be ms-vscs-rm ... the publisher registered on Marketplace with a display name as Microsoft

File: Extensions/ServiceNow/Src/vss-extension.json [hunk 8]
```diff
@@ -4,8 +4,8 @@
   "name": "ServiceNow change request management",
```
Comment (shashban) [inline Extensions/ServiceNow/Src/vss-extension.json]: the id should be changed so it doesn't conflict with the private preview extension

**Pull request merged**